generator/.DevConfigs/9fa1d027-8266-430e-b4c0-a541eb733f8e.json

```diff
@@ -0,0 +1,18 @@
+{
+    "core": {
+        "changeLogMessages": [
+            "Updated credential providers to provide Async version for generating credentials."
+        ],
+        "type": "patch",
+        "updateMinimum": false
```

Comment:
It doesn't matter here for V4 since we ship all packages in V4, but this should be `true` (it's rare that a change to Core won't require all services to be updated too).

Reply:
Fixed.